optuna/logging.py

```diff
@@ -127,18 +127,32 @@ def disable_default_handler():
 
     Example:
 
-        Stop and then resume logging to standard output.
+        Stop and then resume logging to stderr.
 
-        .. code::
+        .. testsetup::
+
+            def objective(trial):
+                x = trial.suggest_uniform('x', -100, 100)
+                y = trial.suggest_categorical('y', [-1, 0, 1])
+                return x ** 2 + y
+
+        .. testcode::
+
+            import optuna
+
+            study = optuna.create_study()
+
+            # There are no logs in stderr.
```

Comment:
This is a trivial comment, but I think it would be helpful if the full module path is shown here.
```suggestion
            # There are no logs in sys.stderr.
```